lib/active_merchant/billing/gateways/pay_gate_xml.rb

```diff
@@ -251,6 +270,13 @@ def commit(action, request)
           :authorization  => response[:tid]
         )
       end
+      def commit_capture(action, authorization, request)
```

Comment:
You can simply have an extra param in ``` commit ``` called auth, if present use that instead of ``` response[:tid] ```